templates/external_auth_api.conf.erb

```diff
@@ -0,0 +1,20 @@
+
+<LocationMatch ^/api(/v2)?/users/extlogin/?$>
+  SSLRequireSSL
+  <% if scope.lookupvar('$foreman::ipa_authentication_api') %>
+  AuthType GSSAPI
+  AuthName "GSSAPI Single Sign On Login"
+  GssapiCredStore keytab:<%= @http_keytab %>
+  GssapiSSLonly On
+  GssapiLocalName <%= @gssapi_local_name %>
+  <% else %>
```

Comment:
Ruby's `if` statements would enable GSSAPI **or** Basic auth **only**, which is not quite right. The previous Apache's module `if` statement must remain since it allow us to use either negotiation or credentials for the auth depending on the request headers.